<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuSystem</class>
 <widget class="QDialog" name="MenuSystem">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: ./images/bench_icon.png
</string>
   </property>
   <widget class="QLabel" name="background_sys">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>154</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_menu">
     <item>
      <widget class="QCommandLinkButton" name="btn_syst">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_graph">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_bench">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_net">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>671</width>
      <height>247</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layout_perf">
     <item>
      <widget class="QQuickWidget" name="radial_perf">
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
       <property name="source">
        <url>
         <string>qrc:/external/RadialBarDemo/radialbar.qml</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_sysInfo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>System Info :</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layout_sysInfo">
     <item row="1" column="0">
      <widget class="QLabel" name="label_sysLinux">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Linux version</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="txt_os">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_sysKernel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kernel version</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_sysUser">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="txt_user">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="txt_kernel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layout_cpuInfo">
     <item row="0" column="0">
      <widget class="QLabel" name="label_cpuModel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_cpuCore">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Core</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="txt_model">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="txt_core">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_cpuSpeed">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="txt_speed">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_cpuInfo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU Info :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
